UnitySetup/UnitySetup.psm1

```diff
@@ -318,7 +319,8 @@ function Find-UnitySetupInstaller {
         [UnitySetupComponent]::Facebook = , "$targetSupport/UnitySetup-Facebook-Games-Support-for-Editor-$Version.$installerExtension";
         [UnitySetupComponent]::Linux = , "$targetSupport/UnitySetup-Linux-Support-for-Editor-$Version.$installerExtension";
         [UnitySetupComponent]::Mac = "$targetSupport/UnitySetup-Mac-Support-for-Editor-$Version.$installerExtension",
-        "$targetSupport/UnitySetup-Mac-Mono-Support-for-Editor-$Version.$installerExtension";
+         "$targetSupport/UnitySetup-Mac-Mono-Support-for-Editor-$Version.$installerExtension";
```

Comment:
Are these the same component and they renamed it? Or can Mac-Mono-Support and Mac-Support exist side by side?

Reply:
nothing is renamed, not sure why that line changed, looks like just an extra space on the end. I can revert it, but it seemed to not have been formatted correctly or something.